sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/FeatureFlagSettingUnitTest.java

```diff
@@ -134,9 +134,9 @@ private String getUnknownPropertiesFeatureFlagConfigurationSettingValue(String i
         String displayName, boolean isEnabled) {
         return String.format("{\"id\":\"%s\",\"additional_field_1\":\"additional_value_1\",\"description\":\"%s\",\"display_name\":\"%s\",\"enabled\":%s,"
                 + "\"conditions\":{\"requirement_type\":\"All\",\"client_filters\":"
-                + "[{\"name\":\"Microsoft.Percentage\",\"parameters\":{\"Value\":\"30\"}}]"
+                + "[{\"name\":\"Microsoft.Percentage\",\"parameters\":{\"Value\":30}}]"
```

Comment:
change String value to numerical value is because AppConfig portal accepts numerical value and then it is a correct feature flag json